net.sf.eclipsecs.core/src/net/sf/eclipsecs/core/config/meta/CheckUtil.java

```diff
@@ -0,0 +1,84 @@
+package net.sf.eclipsecs.core.config.meta;
+
+import java.util.Set;
+
+import com.puppycrawl.tools.checkstyle.PackageNamesLoader;
+import com.puppycrawl.tools.checkstyle.PackageObjectFactory;
+import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
+import com.puppycrawl.tools.checkstyle.api.CheckstyleException;
+import com.puppycrawl.tools.checkstyle.checks.javadoc.AbstractJavadocCheck;
+import com.puppycrawl.tools.checkstyle.utils.JavadocUtil;
+import com.puppycrawl.tools.checkstyle.utils.TokenUtil;
+
+import net.sf.eclipsecs.core.CheckstylePlugin;
+
+public final class CheckUtil {
+    private CheckUtil() {
+    }
+
+    public static Object getCheck(String checkName) {
```

Comment:
`public abstract class AbstractJavadocCheck extends AbstractCheck {`

as you claim to load Check, please do cast to AbstractCheck in this method.

Reply:
Done